package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "rollup-plugin-postcss",
-  "version": "3.0.0",
+  "version": "3.1.8",
```

Comment:
Do you need to bump version manually? 3.1.8 was published in August, I assume next release including these changes will actually have a different version